main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	setup "./src/setup"
+)
```

Comment:
Importing with relative path is uncommon now. Please use go modules for future stability.
see: https://qiita.com/momotaro98/items/23fa4356389a7e610acc

TL;DR: create go.mod and import local modules with absolute path (e.g.: `setup github.com/approvers/go-vcnotify/src/setup`